examples/s/d/nn/xcase/realization-0/iter-0/rms/bin/export_propmaps.py

```diff
@@ -34,11 +34,10 @@
 
 INPUT_FOLDER = Path("../output/maps/grid_averages")
 
-dataio.ExportData._inside_rms = True
```

Comment:
I would assume people do not import this example into there prod. code. But who knows, feels sketchy to set this flag on import.

Reply:
This example is emulating being ran inside RMS, I believe. So this should never be used like this in production code. I don't think the likelihood of that happening is particularly significant but Murphys's Law tells us it will happen. Not sure what other options there are, but perhaps instead of setting this we could manipulate the environment so that fmu-dataio believes it is inside RMS instead.